examples/cassandra/java/src/io/k8s/cassandra/KubernetesSeedProvider.java

```diff
@@ -64,7 +64,7 @@ public KubernetesSeedProvider(Map<String, String> params) {
 
         String host = protocol + "://" + hostName + ":" + hostPort;
         String serviceName = getEnvOrDefault("CASSANDRA_SERVICE", "cassandra");
-        String path = "/api/v1beta1/endpoints/";
+        String path = "/api/v1beta3/endpoints/";
```

Comment:
I ran through the tutorial, but I didn't check if Cassandra worked properly. I submitted a PR for rethinkdb before I left today. I will ask @nikhiljindal to hold that PR.